server/kotc_database/src/lib.rs

```diff
@@ -33,7 +32,10 @@ pub async unsafe fn establish_connection() {
     match VAL {
         Some(_) => (),
         None => {
-            let database_url = "postgres://postgres:password@db/kotc".to_string();
+            let database_url = match option_env!("DATABASE_URL") {
+                Some(url) => url.to_string(),
+                _ => "postgres://postgres:password@db/kotc".to_string(),
```

Comment:
Perhaps it would be better to panic when env var is not set? Easier to debug what's going on